packages/server/src/createServerAdapter.ts

```diff
@@ -216,3 +216,26 @@ function createServerAdapter<
 }
 
 export { createServerAdapter };
+
+// from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#copying_accessors
```

Comment:
Could we do a [structured clone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) here?

Reply:
I do not think so, because at some point we have to merge multiple objects together, we're not really "cloning".